beacon-chain/p2p/subnets.go

```diff
@@ -131,7 +133,10 @@ func (s *Service) filterPeerForSyncSubnet(index uint64) func(node *enode.Node) b
 // for a subnet. So that even in the event of poor peer
 // connectivity, we can still broadcast an attestation.
 func (s *Service) hasPeerWithSubnet(topic string) bool {
-	return len(s.pubsub.ListPeers(topic+s.Encoding().ProtocolSuffix())) >= 1
+	// In the event peer threshold is lower, we will choose the lower
+	// threshold.
+	minPeers := mathutil.Min(1, flags.Get().MinimumPeersPerSubnet)
```

Comment:
Shouldn't this be mathutil.Max? Otherwise minPeers will always be 0 or 1.
```suggestion
	minPeers := mathutil.Max(1, flags.Get().MinimumPeersPerSubnet)
```

Reply:
Previously the limit was `1`, we just want to account for the case where a user is fine with there being 0 peers in the subnet. The default value of  `flags.Get().MinimumPeersPerSubnet` is 6.